src/java/edu/ucsd/library/dams/api/FedoraAPIServlet.java

```diff
@@ -1033,12 +1033,22 @@ else if ( parentColVisibility.equals(VISIBILITY_LOCAL) )
 		{
 			accessGroup = roleAdmin;
 		}
+		else if ( visibility != null && visibility.equals("public") )
```

Comment:
There are four `visibility != null` checks in a row. Would there be any harm in just setting visibility to "" earlier in the function to avoid the null checks?

Reply:
@mcritchlow Sure thing and it's done.